skare3_tools/packages.py

```diff
@@ -827,9 +849,6 @@ def _get_repository_info_v3(
     :param since: int or str
         the maximum number of releases to look back, or the release tag to look back to
         (not inclusive).
-    :param use_pr_titles: bool
```

Comment:
This isn't user-facing but maybe add to the description that this basically-unused feature is removed with this PR.